src_modules/voice_commands/phrases.ts

```diff
@@ -0,0 +1,17 @@
+import { SimplePhrase } from "extcord";
+
+export const voiceCommandsEnabledPhrase = new SimplePhrase({
```

Comment:
Use MessagePhrase instead of SimplePhrase in all of these. MessagePhrases are used for message responses and support embeds, simple phrases are simply a translation of a text.